config/environments/impersonate.rb

```diff
@@ -1,29 +0,0 @@
-require Rails.root.join("config/environments/development")
```

Comment:
All the settings in this file are now handled by `config/environment/development`, because it pulls in data from `config/secrets`.  So the script in `script/setup_impersonate` can simply copy the development into a node-specific environment.  Hence, the `.gitignore` can now ignore all of the impersonate environment files and this file is removed.  This file was actually serving as a wrapper for node-specific environment files.